src/core/storage/preferences/preferencesStorage.ts

```diff
@@ -13,6 +13,7 @@ enum PreferencesKeys {
   APP_HIDE_CRYPTO_BALANCES = "app-hide-crypto-balances",
   APP_IDENTIFIERS_FAVOURITES = "app-identifiers-favourites",
   APP_CREDS_FAVOURITES = "app-creds-favourites",
+  APP_MEERKAT_SEED = "app-meerkat-seed",
```

Comment:
This should use the secure storage, and not the preferences storage. Not sure how I missed that